### PR TITLE
HPCC-31558 Create HPCC-Platform content for ECL Watch UI testing project

### PR DESCRIPTION
**Pull request by AttilaVamos**
- Run regression Setup, then some spray queries to generate content in Platform
- Extract Workunits, Files and DFU Workunits into separated JSON files

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: .github/workflows/test-ui-gh_runner.yml [hunk 1]
```diff
@@ -123,10 +159,17 @@ jobs:
           [[ $retCode -ne 0 ]] && exit 1
           popd
 
+      - name: eclwatch-ui-test-content-artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ inputs.asset-name }}-ui_test-content
+          path: /home/runner/HPCCSystems-regression/log/*.json
+          if-no-files-found: error
+
```
Comment (GordonSmith) [inline .github/workflows/test-ui-gh_runner.yml:168]: Typically we only upload the files on error or cancellation?

File: .github/workflows/test-ui-gh_runner.yml [hunk 2]
```diff
@@ -95,6 +104,33 @@ jobs:
...
+           # In this stage we don't care about the test results
+
+      - name: Get content
+        working-directory:  /home/runner/HPCCSystems-regression/log
+        shell: "bash"
+        run: |
+                curl localhost:8010/WsWorkunits/WUQuery.json | python3 -m json.tool > workunits.json
+                curl localhost:8010/WsDfu/DFUQuery.json?PageSize=250 | python3 -m json.tool > files.json
+                curl localhost:8010/FileSpray/GetDFUWorkunits.json | python3 -m json.tool > dfu-workunits.json
+                ls -l *.json
```
Comment (GordonSmith) [inline .github/workflows/test-ui-gh_runner.yml:132]: Is the `ls` needed?

**Review by GordonSmith — COMMENTED** [review]
Just a couple of questions

File: .github/workflows/test-ui-gh_runner.yml (hunk 1 above)
Comment (AttilaVamos) [reply .github/workflows/test-ui-gh_runner.yml]: Right, changed

File: .github/workflows/test-ui-gh_runner.yml (hunk 2 above)
Comment (AttilaVamos) [reply .github/workflows/test-ui-gh_runner.yml]: Not really, removed.

**Review by GordonSmith — APPROVED** [review]

**Comment by ghalliday** [comment]
@AttilaVamos please squash

**Comment by AttilaVamos** [comment]
@ghalliday done, thanks

**Pull request merged**